02.monte-carlo/Monte_Carlo.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<p>this is a change.</p>
<br/>

 _Reply via <a href='http://127.0.0.1:8000/test-v1/DeepReinforcementLearningNanodegree/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>